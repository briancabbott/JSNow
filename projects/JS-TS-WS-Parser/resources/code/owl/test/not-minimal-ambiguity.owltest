---a = [ '(' ([ '(' ')' ] | 'a' 'b' 'c') ')' ] x*
x = ''
