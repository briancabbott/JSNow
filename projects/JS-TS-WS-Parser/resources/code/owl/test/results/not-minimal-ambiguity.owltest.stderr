error: this grammar is ambiguous

. ( a b c ) 

  can be parsed in two different ways: as

. ( a b c ) 
  a-------- 

  or as

. ( a b c )  
  |        x 
  a--------- 

